### ISSUE TITLE
issue with inlined comments

### ISSUE THREAD

**Issue by gocarlos** [cheshirekow/cmake_format#89]
consider this: 

```cmake
target_link_libraries(${PROJECT_NAME}_lib
                      PRIVATE prometheus-cpp::core
                              prometheus-cpp::pull
                              prometheus-cpp::push
                              # systemx_definitions
                              )
```
run formatter and we get this which is a syntax error: 

```cmake 
target_link_libraries(${PROJECT_NAME}_lib
                      PRIVATE prometheus-cpp::core
                              prometheus-cpp::pull
                              prometheus-cpp::push
                              #systemx_definitions)
```

**Comment by gocarlos** [comment]
I can confirm that this issue is solved in the version 0.4.5, had to upgrade... 
thanks

closing...

**Comment by cheshirekow** [comment]
Great, I'm glad to hear that it's working for you now. Thanks for following up.
